storage/range.go

```diff
@@ -113,7 +114,7 @@ type configDescriptor struct {
 
 // configDescriptors is a slice containing the accounting, permissions
```

Comment:
Because I wanted a constant-sized array for the checksums each range keeps track of.